feat: cleanup text for submission (#62)
lamalab-org/chembench-paper: src/tex/ms.tex

\clearpage
\begin{abstract}
Large language models (LLMs) have gained widespread interest due to their ability to process human language and perform tasks on which they have not been explicitly trained.

However, we possess only a limited systematic understanding of the chemical capabilities of LLMs, which would be required to improve models and mitigate potential harms.
Here, we introduce \enquote{\chembench,} an automated framework for evaluating the chemical knowledge and reasoning abilities of state-of-the-art LLMs against the expertise of chemists.

> We curated more than 2,800 question-answer pairs, evaluated leading open—and closed-source LLMs, and found that the best models outperformed the best human chemists in our study on average.

Reviewer: ```suggestion
We curated more than 2,800 question-answer pairs, evaluated leading open- and closed-source LLMs, and found that the best models outperformed the best human chemists in our study on average.
```

> However, the models struggle with some tasks that are easy for human experts and provide overconfident predictions.

Reviewer: ```suggestion
However, the models struggle with some tasks that are easy for human experts, providing overconfident predictions.
```

Author: hm -  what is the motivation behind this suggestion? I felt that the "and" was here for the enumeration of two separate observations. Without "and" it seems a bit more as if "overconfident predictions" would be one of the tasks that are easy for humans

Reviewer: Sounded better to me, but with the "and" sounds also good

To compile our benchmark corpus, we utilized a broad list of sources (see \Cref{sec:curation}), ranging from completely novel manually crafted questions, over university exams to semi-automatically generated questions based on curated subsets of data in chemical databases.

> For quality assurance, all questions have been reviewed by at least one scientist in addition to the original curator and automated checks.

Reviewer: For these second round we implemented the two approvals method, so I do not know if we should change this:

```suggestion
For quality assurance, all questions have been reviewed by at least two scientists in addition to the original curator and automated checks.
```

    \centering
    \includegraphics{figures/human_subset_performance.pdf}

>     \caption{\textbf{Performance of models and humans on the \enquote{tiny} subset of the  \chembench corpus.} The figure shows the percentage of questions that the models answered correctly. We use horizontal bars to indicate the performance of various models and highlight statistics of the human performance.

Reviewer: ```suggestion
    \caption{\textbf{Performance of models and humans on the \enquote{human} subset of the  \chembench corpus.} The figure shows the percentage of questions that the models answered correctly. We use horizontal bars to indicate the performance of various models and highlight statistics of the human performance.
```

Author: not sure what nomenclature we should use - we call it "tiny" in other places (with reference to the "tiny benchmarks" paper)

Author: > I'd go for human everywhere, since the purpose of that subset was to evaluate the humans

yes and no. The purpose also was to to really have this smaller subset you can run during the training. This is not feasible with the complete ChemBench corpus.

This indicates that there is still room for improvement of chemical \gls{llm} by training them on more specialized data sources or by integrating them with specialized databases.

In addition, our analysis shows that the performance of models is correlated with their size (see \Cref{fig:model_size_plot}). This is in line with observations in other domains but also indicates that chemical \gls{llm} could be further improved by scaling them up.

\paragraph{Performance per topic} To obtain a more detailed understanding of the performance of the models, we also analyzed the performance of the models in different subfields of the chemical sciences.
For this analysis, we defined a set of topics (see \Cref{sec:meth-topic}) and  classified all questions in the \chembench corpus into these topics.
We then computed the percentage of questions the models or humans answered correctly for each topic and show them in \Cref{fig:all_questions_models_completely_correct_radar_human}.

> In this spider chart, the worst score for every dimension is zero (no question answered correctly), and the best score is one (all questions answered correctly). Thus, a larger colored area indicates a better performance.

Reviewer: Perhaps, I would move this to the caption of the figure

Author: Yes, this used to be my intuition as well. However, I learned from successful paper writers that explaining figures in the text can be beneficial if you want it to be one continuous narrative.

\end{figure}

One can observe that this performance varies across models and topics.
While general and technical receive relatively high scores for many models, this is not the case for topics such as toxicity and safety or analytical chemistry.

In the subfield of analytical chemistry the prediction of the number of signals observable in a \gls{nmr} spectrum proved difficult for even for the best models (e.g., \variable{output/subset_scores/is_number_nmr_peaks_o1.txt} percent correct answers for \oone).
Importantly, while the human experts are given a drawing of the compounds, the models are only shown the \gls{smiles} string of a compound and have to use this to reason about the symmetry of the compound (i.e., to identify the number of diasterotopically distinct protons, which requires \emph{reasoning} about the topology and structure of a molecule).
 These findings also shine an interesting light on the value of textbook-inspired questions.
 A subset of the questions in the \chembench are based on textbooks targeted at undergraduate students.

>  On those questions, the models tend to perform better than on some of our semi-automatically constructed tasks (see \Cref{fig:performance_per_topic}).

Reviewer: These sentences might originate comments in the review since the size of subset are not very comparable (~1300 semi auto questions vs 20 textbook questions)

Author: what do you suggest? 

I think the textbook question count you have might be off - the entire `oup` directory is text-book questions.

Reviewer: I am correcting the oup thing, and then with the new count, probably is already consistent enough

\paragraph{Judging chemical preference}

One interesting finding of recent research is that foundation models can judge interestingness.\autocite{zhang2024omniopenendednessmodelshuman, Argyle_2023}
If models could do so for chemical compounds, this would open opportunities for novel optimization approaches.
Such open-ended tasks, however, depend on an external observer defining what interestingness is.\autocite{hughes2024openendednessessentialartificialsuperhuman}

> Here, we posed models the same question~\textcite{Choung_2023} asked chemists at a drug company: \enquote{Which of the two compounds do you prefer?} (in the context of an early virtual screening campaign setting).

Reviewer: Maybe add a reference to the chembench examples table?

Author: yep - do you want to add a suggested change we can directly commit here?

> Here, we posed models the same question~\textcite{Choung_2023} asked chemists at a drug company: \enquote{Which of the two compounds do you prefer?} (in the context of an early virtual screening campaign setting).

Reviewer: ```suggestion
Here, we posed models the same question~\textcite{Choung_2023} asked chemists at a drug company: \enquote{Which of the two compounds do you prefer?} (in the context of an early virtual screening campaign setting) (see \Cref{tab:chembench_corpus_cognitive}).
```
